src/lib/operators/join_hash.cpp

```diff
@@ -466,6 +483,18 @@ class JoinHash::JoinHashImpl : public AbstractJoinOperatorImpl {
             pos_list_left[current_partition_id] = pos_list_left_local;
             pos_list_right[current_partition_id] = pos_list_right_local;
           }
+        } else if (_mode == JoinMode::Anti) {
```

Comment:
I'm missing something. How can the `anti` case work without any `if` statement? Where do we actually check something?

Reply:
This is still an `else` branch of the original `if (hashtables)`. I will re-organize the probing code a bit to clear things up